src/effects/VST3/VST3Utils.cpp

```diff
@@ -122,6 +122,27 @@ bool VST3Utils::ParseAutomationParameterKey(const wxString& key, Steinberg::Vst:
    return false;
 }
 
+wxString VST3Utils::MakeFactoryPresetID(Steinberg::Vst::UnitID unitId, Steinberg::int32 programIndex)
+{
+   return wxString::Format("%d:%d",
```

Comment:
Are you certain ':' will not occur in either of the substituted strings?  The inverse function assumes this.

If you doubt it, maybe the utility functions `wxSplit` and `wxJoin` can be used, to easily escape  certain characters.